superset-frontend/cypress-base/package.json

```diff
@@ -11,6 +11,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cypress/code-coverage": "^3.9.2",
+    "@superset-ui/core": "^0.17.18",
+    "react-dom": "^16.13.0",
```

Comment:
Needed both for typing.. should not actually increase build speed or bundle size.